code/src/Albums.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Artist } from './Artists'
+import { HoverIcons } from 'Hovericons'
+
+
+// This compontent will fetch all data related to the Album including Album name and album image.
+export const Album = (props) => {
+  return (
+    <article className='album-card'>
+      <div className="Img-wrapper">
```

Comment:
You seem to use `'` everywhere else so it's good if you follow the same here (and check my comment before this one about the big letter)
```suggestion
      <div className='img-wrapper'>
```